homeassistant/components/touchline/manifest.json

```diff
@@ -2,9 +2,10 @@
   "domain": "touchline",
   "name": "Roth Touchline",
   "codeowners": [],
+  "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/touchline",
+  "integration_type": "hub",
   "iot_class": "local_polling",
   "loggers": ["pytouchline"],
-  "quality_scale": "legacy",
-  "requirements": ["pytouchline==0.7"]
+  "requirements": ["pytouchline_extended==0.4.5"]
```

Comment:
Can we look into migrating to this library in a preliminary PR? This way we scope that PR as "change from lib A to B" and scope this PR into "add new config method"

Reply:
I see you merged the other PR, so i think this should be okay ?